fluentui_drawer/src/main/java/com/microsoft/fluentui/tokenized/drawer/Drawer.kt

```diff
@@ -123,20 +140,35 @@ class DrawerState(
     suspend fun open() {
         enable = true
         animationInProgress = true
-        do{
+        do {
             delay(50)
-        }while(!anchorsFilled)
+        } while (!anchorsFilled)
+        /*
+        * open and expanded states are possible in case of
```

Comment:
rephrase the comment for clarity

Reply:
I have changed comments, and descriptions for better clarity.